src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/medical/Vitamins.java

```diff
@@ -33,6 +34,7 @@ public ItemUseHandler getItemHandler() {
             e.cancel();
             p.setFireTicks(0);
             clearNegativeEffects(p);
+            RadioactivityTask.removePlayer(e.getPlayer());
```

Comment:
Either move this into the `clearNegativeEffects` or at least use the `p` variable.